cicd/index.adoc

```diff
@@ -15,19 +15,19 @@ toc::[]
 
 [id="openshift-builds"]
 == OpenShift Builds
-OpenShift Builds provides you the following option to configure and run a build:
+OpenShift Builds provides you the following options to configure and run a build:
 
-// * Builds using Shipwright
-// +
-// An extensible build framework based on the Shipwright project, which you can use to build container images on an {product-title} cluster. You can build container images from source code and Dockerfile by using image build tools, such as Source-to-Image (S2I) and Buildah.
-// +
-// For more information, see link:https://docs.openshift.com/builds/latest/about/overview-openshift-builds.html[Overview of Builds].
+* Builds using Shipwright
```

Comment:
This technically might be out of scope for this PR, but I think it might be best to structure the bullets for this one and the  "Builds using BuildConfig" one kind of like the ones here https://docs.openshift.com/container-platform/4.14/architecture/architecture.html#architecture-platform-benefits_architecture. Reason I say that, Is there is some similar guidance in the ISG for punctuation https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists because at the very least "Builds using Shipwright" and "Builds using BuildConfig" need a period after them.

Reply:
fixed